modules/check_in/spec/request/v2/appointments_request_spec.rb

```diff
@@ -235,10 +245,72 @@
         end
 
         it 'returns error' do
-          VCR.use_cassette 'check_in/facilities/get_facilities_500' do
-            VCR.use_cassette 'check_in/appointments/get_appointments_200' do
-              VCR.use_cassette 'check_in/map/security_token_service_200' do
-                get "/check_in/v2/sessions/#{id}/appointments", params: { start: start_date, end: end_date }
+          VCR.use_cassette 'check_in/clinics/get_clinics_500' do
+            VCR.use_cassette 'check_in/facilities/get_facilities_500' do
+              VCR.use_cassette 'check_in/appointments/get_appointments_200' do
+                VCR.use_cassette 'check_in/map/security_token_service_200' do
+                  get "/check_in/v2/sessions/#{id}/appointments", params: { start: start_date, end: end_date }
+                end
+              end
+            end
+          end
+
+          expect(response).to have_http_status(:bad_request)
+          expect(response.body).to eq(error_response)
+        end
+      end
+
+      context 'when facility service returns 500 and clinic service succeeds' do
```

Comment:
the way code is written currently, if facilities call fails, it doesn't matter if clinics call passes or fails. so this may be redundant.

Reply:
Please find the updates in [3b0ce1a](https://github.com/department-of-veterans-affairs/vets-api/pull/17642/commits/3b0ce1aae8965571b1f8677904383f8199a65367)